diff --git a/node_modules/vercel/dist/index.js b/node_modules/vercel/dist/index.js
index cb8a7e7..3d3f59d 100755
--- a/node_modules/vercel/dist/index.js
+++ b/node_modules/vercel/dist/index.js
@@ -186524,7 +186524,7 @@ var init_server = __esm({
             req.url = location;
           }
           if (callLevel === 0) {
-            await this.updateBuildMatches(vercelConfig);
+            // await this.updateBuildMatches(vercelConfig);
           }
           if (this.blockingBuildsPromise) {
             debug3("Waiting for builds to complete before handling request");
@@ -187399,7 +187399,8 @@ Please ensure that ${cmd(err.path)} is properly installed`;
         return void 0;
       }
       async _getVercelConfig() {
-        const configPath = getLocalPathConfig(this.cwd);
+        const originalCwd = process.cwd()
+        const configPath = getLocalPathConfig(originalCwd);
         const [
           pkg = null,
           // The default empty `vercel.json` is used to serve all
